Resources/views/Form/formatter.html.twig

```diff
@@ -49,6 +49,14 @@
                         appendClass = 'html';
                         break;
                     case 'richhtml':
+                        {%- for name, plugin in ckeditor_plugins if ckeditor_plugins is defined -%}
```

Comment:
Cannot we add the plugin to the current instance, and not the global configuration ? if the configuration is global then 2 ckeditor on the same page will share the some set of plugins,.